test/test_xfft.cpp

```diff
@@ -0,0 +1,86 @@
+#include "test_common_macros.hpp"
+#include "xtensor/xmath.hpp"
+#include "xtensor/xarray.hpp"
+#include "xtensor/xfft.hpp"
```

Comment:
Do you see a way to have some tests based on random data comparing to FFTW?

Reply:
This would be solved too with the API suggested by @JohanMabille

Reply:
I don't want to make the library depend on fftw is the only problem...